src/Nox.Types/Types/Color/Color.cs

```diff
@@ -0,0 +1,213 @@
+﻿using System;
+using System.Globalization;
+using System.Linq;
+
+namespace Nox.Types;
+
+/// <summary>
+/// Represents a Nox <see cref="Color"/> type and value object. 
+/// </summary>
+public class Color : ValueObject<byte[], Color>
+{
+    private byte _alpha { get; set; }
+    private byte _red { get; set; }
+    private byte _green { get; set; }
+    private byte _blue { get; set; }
+    
+    /// <summary>
+    /// Creates a new instance of <see cref="Color"/> object.
+    /// </summary>
+    public Color()
+    {
+        Value = new byte[4];
+    }
+
+    /// <summary>
+    /// Creates a new instance of <see cref="Color"/> object with empty color.
+    /// </summary>
+    public static Color Empty
```

Comment:
We do not need this static property, because we can always instantiate an empty color by:
```
var color = new Nox.Types.Color();
or
var emptyColor = new Nox.Types.Color().From(System.Drawing.Color.Empty)
```

Reply:
Done